src/definitions.ts

```diff
@@ -404,9 +408,32 @@ export interface CapacitorCalendarPlugin {
    * console.log(result.failed) // ['ID_DOES_NOT_EXIST']
    */
   deleteEventsById(options: {
-    ids: string[];
+    ids: string[]
   }): Promise<{ result: { deleted: string[]; failed: string[] } }>;
 
+  /**
+   * Deletes an even from the calendar by their ID.
+   *
+   * @async
+   * @since TODO: Add version number
+   * @platform iOS, Android
+   * @permissions
+   * <h3>Runtime Permissions:</h3>
+   * <ul>
+   *   <li><strong>iOS:</strong> writeCalendar</li>
+   *   <li><strong>Android:</strong> writeCalendar</li>
+   * </ul>
+   * @param {object} options Options for defining event ID and span.
+   * @param {number} options.ids The ID of the event that should be deleted.
+   * @returns {Promise<{ result: string }>}
+   * A promise that resolves to an object with one property:
+   *  - result: string - The ID of the deleted event.
+   * @example
+   * const { result } = await CapacitorCalendar.deleteEventsById("ID_1")
+   * console.log(result.deleted)  // ['ID_1']
+   */
+  deleteEventById(options: {id: string, span?: EventSpan}): Promise<{ result: string }>;
```

Comment:
This method could return `Promise<void>`. Since only one event is requested to be deleted and the id is already known by the developer so returning void is enough to indicate the deletion was successful.